src/lib/y2storage/yaml_writer.rb

```diff
@@ -440,5 +441,56 @@ def yaml_btrfs_subvolume(subvol)
       content["nocow"] = "true" if subvol.nocow?
       { "subvolume" => content }
     end
+
+    # Return YAML for one unsupported device.
+    #
+    # @param  device [Device]
```

Comment:
Repeating a comment that got lost in the push.

It's maybe worth mentioning that it must be a device responding to `#name`. BTW, in the usual Yardoc/Ruby notation that can be expressed like:
```ruby
# @param device [#name]
```